app/(afterLogin)/(payroll)/employee-information/[id]/page.tsx

```
@@ -246,375 +248,406 @@ const EmployeeProfile = () => {
+                        style={{
+                          display: 'flex',
+                          flexDirection: 'column',
+                          gap: 16,
+                        }}
+                      >
+                        {/* Base Salary Row */}
+                        <div className="flex flex-col md:flex-row gap-2">
+                          <Text className="min-w-[120px]">Base Salary:</Text>
+                          <Text strong>
+                            {
+                              activeMergedPayroll?.employeeInfo?.basicSalaries[0]
```

ESLint

[error] Delete `⏎······························`

(prettier/prettier)